about.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="stylesheet" href="about.css">
+  <link rel="stylesheet" href="//fonts.googleapis.com/earlyaccess/nanumgothic.css">
+
+</head>
+<body>
+  <main class="maincontainer">
+    <header class="header">
+      <ul class="header1">
+        <li><img src="./images/facebook.png" alt="Facebook"></li>
+        <li><img src="./images/twitter.png" alt="Twitter"></li>
+        <li>Korean</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+      <div class="header2">
+        <div class="header25">
+          <nav class="dropdown">
+            <button type="button" id="button" class="button">
+              <img src=".images/hamburguer.png" alt="Menu" onclick="dropmenu()">
```

Comment:
- [ ] Kindly try to fix the path of the hamburger menu on the about page because now the image does not display
![image](https://user-images.githubusercontent.com/24703171/169205701-3033c5ce-6fe6-4e4f-b972-185e23342d65.png)